app/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,6 @@
     <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
 
     <uses-permission
-        android:requiredFeature="@string/ark_file_picker_pick"
```

Comment:
This is for fixing a side effect when files access permission toggle was disabled, caused by this commit: c7d14c1ac7d17ed9d0191752dbe67bd4b213467a